data/tutorials/gs_00_up_and_running.md

```diff
@@ -36,27 +36,27 @@ port install opam
 
 **For Linux** the preferred way is to use your system's package manager on
 Linux (e.g `apt-get install opam` or similar). [Details of all installation
-methods.](https://opam.ocaml.org/doc/Install.html)
+methods.](https://opam.ocaml.org/doc/Install.html). All supported Linux distributions package at least version 2.0.0 (you can check by running `opam --version`). If you are using an unsupported Linux distribution, please either download a precompiled binary or build opam from sources.
```

Comment:
Should this be in the "For Linux" category? My gut feeling is that the main source of severely outdated opam would be a virtual image of an old system, can't MacOS + brew have the same issue?

(Reading this section again, my impression is that the "For macOS / For Linux" split could be part of its own subsection "Install opam" before "Install OCaml", and that this subsection would be the right place to have this warning.)

Reply:
I don't think it applies to macOS. On an out-of-date Debian, assuming `apt-get update` works at all out of the box of the box (eventually the repos get moved to archives), you will get OPAM 1.2.2 because it uses an old repository to go with the old distro. On macOS you get the same repository whichever version of macOS you're running and, even if for some reason opam 2.x wouldn't install, brew and macports only carry one version of a package, so you'd either get the latest version of opam regardless or an error trying to install it.

Reply:
Okay, sounds reasonable.